apps/cmp_mock_srv.c

```diff
@@ -360,8 +360,10 @@ static int process_certConf(OSSL_CMP_SRV_CTX *srv_ctx,
 }
 
 static int process_pollReq(OSSL_CMP_SRV_CTX *srv_ctx,
-                           const OSSL_CMP_MSG *pollReq, int certReqId,
-                           OSSL_CMP_MSG **certReq, int64_t *check_after)
+                           const OSSL_CMP_MSG *pollReq,
+			   ossl_unused int UNUSED_certReqId,
```

Comment:
wrong indentation on 2 lines.

Reply:
It was tab vs. space. All tabs are gone now.